ui/raidboss/data/triggers/o9s.js

```diff
@@ -333,7 +337,7 @@
     },
     {
       id: 'O9S Knock Down Marker',
-      regex: / 1B:........:(\y{Name}):0000:0000:0057:0000:0000:0000:/,
+      regex: / 1B:........:(\y{Name}):0000:E000:0057:0000:0000:0000:/,
```

Comment:
I think these should really be `/ 1B:........:(\y{Name}):....:....:0057:/,`, and the equivalent for the `003E` marker.

The first two things after the name aren't consistent (and so I think it was a mistake to specify them here), and although the last ones are always zeroes, maybe better to underspecify.

Reply:
Honestly, no idea what these stand for but they changed on recent FFXIV plugin update.